node/src/Pos/Util/Monitor.hs

```diff
@@ -0,0 +1,47 @@
+{-# OPTIONS_GHC -fno-warn-name-shadowing #-}
```

Comment:
I find it much better to avoid name shadowing than to suppress warnings. If there are fundamental reasons why it can't be avoided, it should be justified in comments.